chartbeat.php

```diff
@@ -244,6 +244,51 @@ function chartbeat_is_validjson($json_str) {
 	}
 }
 
+function chartbeat_configs() {
+	$cb_configs['domain'] = esc_js( apply_filters( 'chartbeat_config_domain', chartbeat_get_display_url (get_option('home')) ));
```

Comment:
@breauxc or @britice can probably speak to this better, but I believe that's the one thing that the pinger does that the AMP pinger is currently unable to do (and is causing some issues since the pings then go to a different h-key/memoryfly shard).